apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -269,7 +269,7 @@ export class SendMessagePush extends SendMessageBase {
           status: ExecutionDetailsStatusEnum.FAILED,
           isTest: false,
           isRetry: false,
-          raw: JSON.stringify(e),
+          raw: JSON.stringify(e) !== JSON.stringify({}) ? JSON.stringify(e) : JSON.stringify(e.message),
```

Comment:
did you or the issue reported reproduce this error? because I am not sure this will work 🤔

i wonder if this is the expected response
https://firebase.google.com/docs/reference/fcm/rest/v1/FcmError